### PR TITLE
Menubar

### PR DESCRIPTION
**Pull request by danielamoreno699**
### Hamburguer icon displaying effect

In this milestone Project, **We**:

-Added visual effect to the hamburguer icon,
-Added additional hidden div container as an overlayout ,
-Added a js file to select the html elements 
-Added  navigation link  to the navbar list

### CODE REVIEW

**Comment by addod19** [comment]
### Hi @danielamoreno699, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is invalid because the README file is missing the future features section.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -18,25 +18,38 @@
...
+        <a class="close-btn" id="cross-icon" onclick="CloseMenu()" >
+            <span style="font-size: 28px;">&times;</span>
+            <!--<img src="./assets/icons8-close-50.png" alt="" >-->
+        </a>
+        <div class="layout-content" id="layout-content" >
+            <a class="pop-nav" href="./portafolio.html" >Portafolio</a>
+            <a class="pop-nav" href="./about.html" >About</a>
+            <a class="pop-nav" href="./contact.html" >Contact</a>
+        </div>
+    </div>
```
Comment (tresorsawasawa) [inline index.html:44]: - [ ] Kindly make sure that when the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed. To help you just use the `<a>` tag on each navigation option then use the id the `href` attribute and after refer the used `id` on each section. Remember to also remove one of your nav option lists as you repeated it twice
e.g.
```HTML
    <li class="li-navbartool"><a href="#portfoilo">Portfolio</a></li>
    <li class="li-navbartool"><a href="#about">About</a></li>
    ...
```
**IMPORTANT:** When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed should work for both **Mobile** and **Desktop** versions

File: styles.css [hunk 2]
```diff
@@ -70,13 +70,63 @@ body {
...
+.effect-icon {
+  display: flex;
+  justify-self: end;
+}
+
 .main-page {
   background-image: url("./assets/header-shapes\ mobile.png");
   background-repeat: no-repeat;
+
+  /* background-size: cover; */
```
Comment (tresorsawasawa) [inline styles.css:126]: - [ ] Kindly ensure that you remove all commented and unused code lines from your code base to make cleaner and for best practices reasons.

File: README.md [hunk 3]
```diff
@@ -97,6 +97,8 @@ PORTAFOLIO WEB VERSION
```
Comment (tresorsawasawa) [inline README.md:97]: - [ ] Kindly ensure that you added all the contributors of the project with working social media links in the `Authors` section of your README and fill all sections with correct content(e.g.: **Show your support**, **Acknowledgments**,...) to make your documentation professional

File: README.md [hunk 4]
```diff
@@ -58,12 +58,12 @@ After you're finished please remove all the comments and instructions!
...
+PORTAFOLIO WEB & MOBILE VERSION
 
-# 📖 [Portafolio Web Version ] <a name="about-project"></a>
+# 📖 [Portafolio Web & Mobile Version ] <a name="about-project"></a>
 
 
-**[Portafolio web version]**
+**[Portafolio web & mobile version]**
 
 ## 🛠 Built With <a name="built-with"></a>
```
Comment (Shedrack-Sunday) [inline README.md:69]: - [ ] As stated by the previous reviewer, you need all files in the linter directory for `HTML-CSS-JS`. Presently you're missing one fro the pic I attached below. Note that these are files to help you stick to best practices 👍 

## View:
image

File: main.js [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+
+
+    
+    
+
+    
+  
+
+    //const DisplayToggleNav = document.getElementById("navbar")
+
```
Comment (Shedrack-Sunday) [inline main.js:182]: - [ ] Also as suggested by the previous reviewer, kindly ensure that you remove all commented and unused code lines from your code base to make it cleaner and for best practices reasons.

File: index.html [hunk 6]
```diff
@@ -18,25 +18,38 @@
...
-        <ul class="ul-navbartool">
-            <li class="li-navbartool">Portafolio</li>
-            <li class="li-navbartool">About</li>
-            <li class="li-navbartool">Contact</li>
+    <a class="brand" href="" id="brand">My logo</a>
+    <nav class="navbar" id="navbar">
+        <ul class="ul-navbartool" id="ul-demo">
+            <li class="li-navbartool" id="portfolio">Portafolio</li>
+            <li class="li-navbartool" id="about">About</li>
+            <li class="li-navbartool" id="contact">Contact</li>
```
Comment (Shedrack-Sunday) [inline index.html:28]: - [ ] As noted here, please make sure that when the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed. To help you just use the `<a>` tag on each navigation option then use the id the `href` attribute and refer to the user id on each section. Remember to also remove one of your nav option lists as you repeated it twice
e.g.

    ```
    <li class="li-navbartool"><a href="#portfoilo">Portfolio</a></li>
    <li class="li-navbartool"><a href="#about">About</a></li>
    ```
    
 ## Instructions :
 
image

File: README.md [hunk 7]
```diff
@@ -148,8 +150,12 @@ https://www.loom.com/share/a216f873fd7d4276a8e5a62378c82584
...
-<!-- FUTURE FEATURES -->
-- **[Animation]**
+ FUTURE FEATURES 
+ 
+- **[popup modal window for project in mobile and desktop version]**
+- **[validation of form with javaScript code]**
+- **[Inclusion of browser storage]**
+
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (Shedrack-Sunday) [inline README.md:161]: -  [ ] Finally, as suggested by your last reviewer, please complete the sections as pointed out in the image below. These are guidelines to show you how to make our work professional, which is very important:

## View
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @danielamoreno699    👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 



Highlights
✅All linters are checked
✅ Proper PR summary 
  
# Required Changes ♻️

- [x] I must admit you have really put in the effort so far in this project, however, the changes required a minor adjustment to fit the previous reviewers suggestions

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 8]
```diff
@@ -18,25 +18,32 @@
...
-            <li class="li-navbartool">Portafolio</li>
-            <li class="li-navbartool">About</li>
-            <li class="li-navbartool">Contact</li>
+    <a class="brand" href="" id="brand">My logo</a>
+    <nav class="navbar" id="navbar">
+        <ul class="ul-navbartool" id="ul-demo">
+
+            <li class="li-navbartool"> <a href="#portafolio" class="li-navbartool">Portafolio</a></li>
+            <li class="li-navbartool"> <a href="#about" class="li-navbartool">About</a></li>
+            <li class="li-navbartool"> <a href="#contact" class="li-navbartool">Contact</a></li>
```
Comment (Davidosky007) [inline index.html:29]: * Well done working on the changes requested so far 👏 but one required change is still left undone
* when a user clicks on the menu options the mobile menu should disappear **HINT** you could add a conditional statement to hide the drop-down
image
 

https://user-images.githubusercontent.com/52497246/223475191-198a73ab-c0c9-4aea-8800-726adbca9a41.mp4

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @danielamoreno699 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Good looking readme file 👍 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidosky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Zuhaib042 — APPROVED** [review]
Hi @danielamoreno699 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
approve

## Highlights
✅ Good Implementation of suggested changes
✅ Professional PR

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**